<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CapteursClass</class>
 <widget class="QMainWindow" name="CapteursClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capteurs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QLabel" name="nlblConnect">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Status :</string>
    </property>
   </widget>
   <widget class="QLabel" name="nlblDonnees">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>données capteur</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblConnect">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Deconnecte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDConnect">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Deconnection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDonnees">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Collecte</string>
    </property>
   </widget>
   <widget class="QLabel" name="nlblTemp">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Température intérieure :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTemp">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nlblHumid">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidité intérieure :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHumid">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDonnees_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Collecte</string>
    </property>
   </widget>
   <widget class="QLabel" name="nlblHumid_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidité 1 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHumid_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nlblHumid_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidité 2 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHumid_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nlblHumid_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidité 3 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHumid_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nlblDonnees_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>données bacs</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblError">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>562</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Capteurs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDConnect</sender>
   <signal>clicked()</signal>
   <receiver>CapteursClass</receiver>
   <slot>onDisconnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked()</signal>
   <receiver>CapteursClass</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDonnees</sender>
   <signal>clicked()</signal>
   <receiver>CapteursClass</receiver>
   <slot>DonneesSensor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDonnees_2</sender>
   <signal>clicked()</signal>
   <receiver>CapteursClass</receiver>
   <slot>DonneesCapteurs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectButtonClicked()</slot>
  <slot>onDisconnectButtonClicked()</slot>
  <slot>DonneesSensor()</slot>
  <slot>DonneesCapteurs()</slot>
 </slots>
</ui>
